### PR TITLE
A11y improvements for meta attributes

### PR DESCRIPTION
**Pull request by JanAckermann**
## Description
- For a11y the html language attribute will be set dynamically <html **lang="xx"**/>
- For a11y the title will be set automatticaly following the schema : sub item (e.G file) - route (e.g All Files) - general  name (e.g ownCloud)

## Related Issue

- https://jira.owncloud.com/browse/WEB-100
- https://jira.owncloud.com/browse/WEB-134

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Review by pascalwengerter — APPROVED** [review]
You could mention the issues resolved here (#4342, #4338) in the changelog, but otherwise looks fine 🔥

File: packages/web-runtime/src/App.vue [hunk 1]
```diff
@@ -215,8 +215,12 @@ export default {
     }
   },
   watch: {
-    $route() {
-      this.appNavigationVisible = false
+    $route: {
+      immediate: true,
+      handler: function(to) {
+        document.title = this.extractPageTitleFromRoute(to)
+        this.appNavigationVisible = false
```
Comment (kulmann) [inline packages/web-runtime/src/App.vue:222]: Please delete this line. ;-) Doesn't belong in here. If the user is on a screen which is wide enough for having the left sidebar and the files list side by side, we don't want to force him into closing it.

**Review by kulmann — CHANGES_REQUESTED** [review]

**Review by kulmann — APPROVED** [review]
Nice 😍 
Could you rebase please? Might be that the failing test pipeline will be fixed by that.

**Comment by JanAckermann** [comment]
already up to date ....

**Comment by kulmann** [comment]
> already up to date ....

ok, thanks. Then we need to have a look at the failing test.

**Comment by JanAckermann** [comment]
@kulmann restarted ci - green now :rocket:

**Pull request merged**